compose/ui/ui/src/webTest/kotlin/androidx/compose/ui/SelectionContainerTests.kt

```diff
@@ -67,7 +68,7 @@ class SelectionContainerTests : OnCanvasTests {
                     selection = it
                     syncChannel.sendFromScope(it)
                 },
-                children = {
+                children = @Composable {
```

Comment:
Shouldn't the compiler plugin take care of this?

Reply:
I was surprised too. 
Actually, it's indeed better to create a YT ticket and investigate too.

Reply:
added a YT ticket. My guess the reason is that the function we call here is internal and we suppress the call of an internal function. The frontent didn't expect such a call and `@composable` was not added implicitly.